public/app/features/alerting/unified/components/admin/AlertmanagerConfig.tsx

```diff
@@ -84,80 +127,53 @@ export default function AlertmanagerConfig(): JSX.Element {
         dataSources={alertManagers}
       />
       {loadingError && !loading && (
-        <Alert severity="error" title="Error loading Alertmanager configuration">
-          {loadingError.message || 'Unknown error.'}
-        </Alert>
+        <>
+          <Alert severity="error" title="Error loading Alertmanager configuration">
+            {loadingError.message || 'Unknown error.'}
+          </Alert>
+
+          {!isFetchingValidAmConfigs && validAmConfigs && validAmConfigs.length > 0 && (
+            <>
+              <div>Choose a previous working configuration</div>
+
+              <Select
+                className={styles.container}
+                options={validAmConfigsOptions}
+                value={selectedAmConfig}
+                onChange={(value: SelectableValue) => {
+                  // @ts-ignore
```

Comment:
What's the reason for `ts-ignore`?

Reply:
There was a problem with the types used in the dropdown. Fixed it.